TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auton/colorSensor/ColorSensorTest.java

```diff
@@ -0,0 +1,20 @@
+package org.firstinspires.ftc.teamcode.auton.colorSensor;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.ColorSensor;
+
+public class ColorSensorTest extends LinearOpMode {
+    ColorSensor colorSensor;
+
+    public void runOpMode() {
+        colorSensor = hardwareMap.colorSensor.get("color");
+
+        waitForStart();
+
+        while (opModeIsActive()) {
+            //finds amount of red, green, blue, and light (alpha)
+            telemetry.addData("hue: ", colorSensor.argb());
```

Comment:
let's store the reading in a variable and then this should be `telemetry.addData("Hue", reading)`